docs/UserGuide.md

```diff
@@ -9,7 +9,7 @@ AddressBook pi (Abπ) is a **360° all-rounded desktop app for managing contacts
 
 <div style="page-break-after: always;"></div>
 
-## Table of Contents <br/>
+## Table of Contents
```

Comment:
Is the formatting still okay if there is no br tag after Table of Contents?

Reply:
Yep!